src/DragAndDropSupportWidget/widget/DroppableWidget.js

```diff
@@ -122,25 +72,19 @@ require( [
                     });
                 }
             }
-
         },
 
-
         /**
          * Called upon creation of onDropEntity
          *
          * @param mendixObject  The new Mendix object
          */
         onDropObjectCreated : function (mendixObject) {
-
             // console.log("onDropObjectCreated");
-            var draggedReference,
```

Comment:
As per convention, variables are to be defined at the top of the function rather than inline. Why did you change this?

Reply:
see above